Userland/Libraries/LibWeb/Layout/Node.cpp

```diff
@@ -226,6 +226,22 @@ void NodeWithStyle::apply_style(const CSS::StyleProperties& specified_style)
         m_background_image = static_ptr_cast<CSS::ImageStyleValue>(bgimage.value());
     }
 
+    auto border_bottom_left_radius = specified_style.property(CSS::PropertyID::BorderBottomLeftRadius);
+    if (border_bottom_left_radius.has_value())
+        computed_values.set_border_bottom_left_radius(border_bottom_left_radius.value().leak_ref().to_length());
```

Comment:
`leak_ref()` will intentionally leak memory. Use `operator->` instead:
```cpp
border_bottom_left_radius.value()->to_length();
```

Same comment repeated for the other instances below :)